web/scm_helpers.go

```diff
@@ -138,6 +138,26 @@ func updateGroupTeam(ctx context.Context, sc scm.SCM, org *pb.Organization, grou
 	return sc.UpdateTeamMembers(ctx, opt)
 }
 
+func rejectUserFromCourse(ctx context.Context, sc scm.SCM, login string, repo *pb.Repository) error {
+
```

Comment:
Avoid extra blank line here.

Should this func be called `removeUserFromCourse`. It will be easier to make connection between the naming, if we use remove or delete consistently.

Reply:
Done.